modules/katello_content_credential.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2018, Baptiste Agasse <baptiste.agagsse@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: katello_content_credential
+short_description: Create and Manage Katello content credentials
+description:
+  - Create and Manage Katello content credentials
+author: "Baptiste Agasse (@bagasse)"
+requirements:
+  - "nailgun >= 0.31.0"
```

Comment:
Want to change this to 0.32 in anticipation?

Reply:
I've also updated this in product and repository modules as we use content_credential in it too.